portfolio/src/main/webapp/script.js

```diff
@@ -48,7 +68,36 @@ async function getComments() {
   comments.forEach((comment) => {
     commentList.appendChild(createCommentElement(comment));
   });
-} /* eslint-enable no-unused-vars */
+}
+
+/**
+ * Get login status of user from json
+ * @param {JSON} json
+ * @return {Boolean} login status
+ */
+function loggedIn(json) {
+  switch (json.logged) {
+    case 'true':
+      return true;
+    case 'false':
+      return false;
+  }
+}
+
+/**
+ * Create anchor element with attributes
```

Comment:
nit: another case where the comments can be more expressive - what does the json object in the @param contain? And what does "some text" mean? A more expressive comment would be:

```
/**
* Takes a JSON object containing user login status and creates 
* an anchor element with a link that depends on whether the user
* is logged in or not:
*  - If the user is logged in, the link logs the user out.
*  - If the user is logged out, the link prompts the user to log in.
* @param {JSON} json - an object containing user link status.
* @return {Element} a - an anchor element with the behaviour described above.
*/
```

Additionally you can mention that the json object is obtained from the login-status URL as well.